config/default.yml

```diff
@@ -50,7 +50,7 @@ RSpec/BeforeAfterAll:
   StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/BeforeAfterAll
 
 RSpec/ContextWording:
-  Description: "`context` block descriptions should start with 'when', or 'with'."
+  Description: Checks that `context` docstring start with 'when', 'with', or 'without'.
```

Comment:
Isn't proper English, should be either `docstrings` either `starts`. However, it's 80 chars long already.
Do you think it makes sense to replace "Checks" with "Check"?

Engineering tradeoffs. I imagined them differently.